src/Avalonia.Controls/ComboBox.cs

```diff
@@ -76,6 +77,14 @@ public class ComboBox : SelectingItemsControl
         public static readonly StyledProperty<VerticalAlignment> VerticalContentAlignmentProperty =
             ContentControl.VerticalContentAlignmentProperty.AddOwner<ComboBox>();
 
+        /// <summary>
+        /// Defines the <see cref="IsAutoSelectEnabled"/> property.
+        /// </summary>
+        public static readonly StyledProperty<bool> IsAutoSelectEnabledProperty =
```

Comment:
Do we need this property? In WPF and UWP as far as I can see this is just default behavior and not sure what the use-case would be for disabling it?

Reply:
I personally like the idea of disabling it actually. Giving more control of the interactions of a control is never bad IMO.

Reply:
The reason why I created this property; for instance, if all of ComboBox items are editable input control, I think it'll be good idea to disable this functionality. Or maybe we can set the defaut value to true?

Reply:
@grokys The reason I created this property; for instance, if all of ComboBox items are editable input control, I think it'll be good idea to disable this functionality. Or, how about set the defaut value to true?

Reply:
Well if the ComboBox items are editable then those controls should set `e.Handled` on the event and it shouldn't reach the `ComboBox`. However if we feel that it'd be useful to be able to disable this behavior I'd be OK with keeping a property to enable it.

I do think though that the default should be `true` like you say, and `IsAutoSelectEnabled` perhaps isn't descriptive enough?

Looking at WPF it seems this behavior can be configured by setting [`TextSearch.TextPath`](https://wpf.2000things.com/tag/textsearch-textpath/) so looks like the feature there is called "text search". Maybe the property should be called `IsTextSearchEnabled`?

Reply:
@grokys Okay, I agree with that :thumbsup: